pkg/kubelet/kubelet.go

```diff
@@ -786,14 +786,14 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	klet.evictionManager = evictionManager
 	klet.admitHandlers.AddPodAdmitHandler(evictionAdmitHandler)
 
-	// Safe, whitelisted sysctls can always be used as unsafe sysctls in the spec.
+	// Safe, allowlisted sysctls can always be used as unsafe sysctls in the spec.
```

Comment:
```suggestion
	// Safe, allowed sysctls can always be used as unsafe sysctls in the spec.
```

tiny inconsequential nit, but this reads a little better to me.